event-exporter/sinks/stackdriver/sink_factory.go

```diff
@@ -51,6 +53,8 @@ func NewSdSinkFactory() sinks.SinkFactory {
 		maxConcurrency: fs.Int("max-concurrency", defaultMaxConcurrency, "Maximum number of "+
 			"concurrent requests to Stackdriver"),
 		location: fs.String("location", "", "GCE location of the cluster"),
```

Comment:
Note that currently cluster location is available from Metadata Server, similarily to cluster name (i.e. `metadata.InstanceAttributeValue("cluster-location")`. It may be a good moment to deprecate this flag and use metadata server instead.